ReactAndroid/build.gradle

```diff
@@ -286,6 +291,9 @@ android {
 
         buildConfigField("boolean", "IS_INTERNAL_BUILD", "false")
         buildConfigField("int", "EXOPACKAGE_FLAGS", "0")
+
+        resValue "integer", "react_native_dev_server_port", reactNativeDevServerPort("8081")
```

Comment:
This isn't strictly necessary, but useful if you link the ReactNative module directly.